### ISSUE TITLE
DIS: IC in F3charm in NC

### ISSUE THREAD

**Issue by felixhekhorn** [scarrazza/apfel#28]
- we believe the IC contributions for F3charm in the NC case are not present
- it is discussed in the documentation but not implemented in the code:
https://github.com/scarrazza/apfel/blob/fdfd068ecfef44f0b7c439b0a887debf46748c02/src/DIS/IncludeIntrinsicCharm.f#L92-L150
https://github.com/scarrazza/apfel/blob/fdfd068ecfef44f0b7c439b0a887debf46748c02/src/DIS/ComputeDISOperators.f#L469-L488

**Comment by vbertone** [comment]
This is correct. The reason is that F3 in NC scattering is relevant only for large values of Q because the couplings are suppressed by the Z propagator. Therefore IC effects (and heavy-quark mass effects in general) are expected to be negligible in the region where F3 becomes relevant.
